### DISCUSSION TITLE
Welcome to Jira-CSV-importer Discussions!

### DISCUSSION THREAD

**Discussion by Okura66** [Okura66/Jira-CSV-importer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alekseykim23** [comment]
@Okura66 are you able to please provide an example `input.csv` file?

**Reply by Okura66** [reply]
Hello @alekseykim23 , just make sure to correctly respect the issuekey for the ticket update column. If it is empty, it will try to create a ticket!
`
issuekey,summary,description,customfield_xxx,customfield_xxx
`
Normally, the field names work, but as a best practice, it's better to use the customfield IDs as in my example. Some default fields can only be used with their names, for example: "description", "due_date", "summary", "assignee", "labels", "reporter".

And please use dev branch instead of main for now.. 😄 Change log level for debug.
